azavea/climate-change-lab
# Feature/kak/jenkins pipeline#151

## Overview

Switch to Jenkins pipeline to build project. Run tests and linter on Jenkins, reporting any linter warnings. For `develop` and `master` branch builds, also push site to S3, to staging or production site, respectively.

### Notes

Should remove the disabled non-pipeline Jenkins build configuration after this is merged.


## Testing Instructions

Since this is a Jenkins configuration PR, there's not really anything to do to test it locally. Can check out  the build logs [here](http://urbanappsci.internal.azavea.com/job/azavea/job/climate-change-lab/view/change-requests/job/PR-160/) and see the slack notifications produced in the channel.

Closes #151
Closes #148
